k9mail/src/main/java/com/fsck/k9/notification/AuthenticationErrorNotifications.java

```diff
@@ -44,6 +46,10 @@ public void showAuthenticationErrorNotification(Account account, boolean incomin
                 .setStyle(new BigTextStyle().bigText(text))
                 .setVisibility(NotificationCompat.VISIBILITY_PUBLIC);
 
+        if (Build.VERSION.SDK_INT >= 21) {
```

Comment:
The constant isn't defined in NotificationCompat. The IDE notes that it's only available in 21. That's why I used an if.